src/ui/public/notify/notifier.js

```diff
@@ -364,6 +370,41 @@ Notifier.prototype.banner = function (msg, cb) {
   }, cb);
 };
 
+/**
+ * Display a custom message
+ * @param  {Object} config
+ * config = {
+ *   title: 'Some Title here',
+ *   markdown: 'Some markdown content',
+ *   type: 'info',
+ *   customActions: [{
+ *     key: 'next',
+ *     callback: function() { next(); }
+ *   }, {
+ *     key: 'prev',
+ *     callback: function() { prev(); }
+ *   }]
+ * }
+ */
+Notifier.prototype.custom = function (config) {
+  const customActionMax = 2;
+  const mergedConfig = _.assign({}, {
```

Comment:
You don't need to assign to an empty object here, you can just mutate the default settings object.